breaking_changes.md

```diff
@@ -0,0 +1,40 @@
+# Breaking changes
+
+
+The following table describes potential changes and if they require a version bump.
+
+Keep in mind there may be instances where these rules are violated if there is a pressing performance requirement from the monolith. We will try to keep violations to an absolute minimum, and broadcast the changes in advance if possible.
+
+For the purposes of this table, parameter means input parameter, whereas attribute refers to the response object.
+
+
+Change Description                                 |  Version Bump Required
+---------------------------------------------------|-----------------------
+Update description/summary/example                 | &#10007;
+Adding attribute or parameter                      | &#10007;
+Removing optional attribute                        | &#10007;
+Removing required attribute                        | &#10003;
+Removing parameter                                 | &#10007;
+Optional attribute becomes required                | &#10007;
+Required attribute becomes optional                | &#10003;
+Optional parameter becomes required                | &#10003;
+Required parameter becomes optional                | &#10007;
+Changing attribute or parameter name               | &#10003;
+Changing attribute or parameter type<sup>1</sup>   | &#10003;
+Changing attribute format<sup>1</sup>              | &#10007;
+Changing parameter format<sup>1</sup>              | &#10003;
+Changing values in an Enum                         | &#10003;
+Adding values to an Enum                           | &#10007;
+Removing values from an Enum                       | &#10003;
```

Comment:
This depends on which direction the enum is going. If we remove a value from an enum of an esi endpoint return signature, we are narrowing the scope of the returned values, so existing code should handle the new subset fine.

If we remove values from an enum that users send /to/ ESI, then we have the same issue as narrowing the format for an existing paramater.

Reply:
fair enough, I'll update this to clarify:

1) if we stop sending some enum member, non-breaking
2) if we stop accepting some enum member, breaking